modules/oe_content_call_proposals/config/install/field.field.node.oe_call_proposals.oe_call_proposals_deadline.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_call_proposals_deadline
+    - node.type.oe_call_proposals
+  module:
+    - datetime
+id: node.oe_call_proposals.oe_call_proposals_deadline
+field_name: oe_call_proposals_deadline
+entity_type: node
+bundle: oe_call_proposals
+label: 'Deadline date'
+description: "Enter the deadline for submission of applications.\r\nNote: the status of the calls change from open to closed when the last deadline has passed.\r\nDate and time follow the CET timezone."
+required: false
```

Comment:
I implemented the Deadline date requirement using the Constraint validator so that the date value is left empty if the model is `permanent` . If any other model is selected, the validator sets the Deadline field as required. 

Is this a correct implementation?

Reply:
There is "Deadline date - date and time (time is optional)" in the specification https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=437854287. So hour and minutes can be left empty. 

Currently error is shown:
```
The Deadline date (value 1) date is incomplete.
A value must be selected for hour.
```

Reply:
Actually users should just set Hour: 0, Minute: 00. So not sure that "time is optional" requirement is relevant.

Reply:
This is done, @yenyasinn could you please review?

Reply:
Yep indeed, I'll report also here: we discussed about it and optional time does not make sense, unless business provides us with a sensible default to default to. Until then let's ignore this requirement.